### PR TITLE
Added migration for upgrade_versions on rolling_capabilities

### PR DESCRIPTION
**Pull request by joedevivo**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended to check this file into your version control system.
```
Comment (seancribbs) [inline db/schema.rb:12]: Well done.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,18 @@ Want to add a new migration? try `bundle exec rake db:new_migration[MigrationNam
 
 Fun Fact: zsh hates []'s. try adding `setopt nonomatch` to your .zshrc to make it love them, unless you are the yesnomatch type, in which case, run `noglob bundle exec rake db:new_migration[MigrationName]`. Don't say I did't warn you.
 
+### How to test migrations?
+Your best bet is to get the current heroku dataset with `pg_dump`. You need the heroku database url, which you can get with `heroku config` if you have access. If you don't, you shouldn't be doing this.
+
+```
+pg_dump postgres://whatever > heroku.sql
```
Comment (seancribbs) [inline README.md]: Add `-O -x` to the pg_dump command. This omits role/permissions information that doesn't make sense for a dev version.

File: db/migrate/2013010908441357746286_add_version_to_capabilities.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
+class AddVersionToCapabilities < ActiveRecord::Migration
+  def up
+    say_with_time "adding upgrade_version to rolling_capabilities" do
+      @projects = Project.where(:name => %w{riak riak_ee}).all
+      %w{rolling_capabilities}.each do |name|
```
Comment (seancribbs) [inline db/migrate/2013010908441357746286_add_version_to_capabilities.rb:5]: Is it really necessary to make this a loop?

**Comment by joedevivo** [comment]
done, added some line breaks too

File: db/migrate/2013010908441357746286_add_version_to_capabilities.rb (hunk 3 above)
Comment (joedevivo) [inline db/migrate/2013010908441357746286_add_version_to_capabilities.rb:5]: nope, not even a little necessary, but I totally copied it from another migration and the less I changed, the less could go wrong.

Comment (seancribbs) [inline db/migrate/2013010908441357746286_add_version_to_capabilities.rb:5]: Not significant, really. Perhaps we should do a deeper review at some point and pull these into a library file.

**Comment by seancribbs** [comment]
The migrations mostly come out at night, mostly.

> The migrations mostly come out at night, mostly.

:+1:

**Comment by cmeiklejohn** [comment]
@seancribbs is on point today.

**Pull request merged**